docs/interpreter/python.md

```diff
@@ -232,6 +232,70 @@ SELECT * FROM rates WHERE age < 40
 Otherwise it can be referred to as `%python.sql`
 
 
+## IPython Support
+
+IPython is more powerful than the default python interpreter with extra functionality. You can use IPython2 or IPython3 which depends on which python you set `zeppelin.python`.
+
+   **Pre-requests**
+   
+    - Jupyter `pip install jupyter`
+    - grpcio `pip install grpcio`
+
+If you already install anaconda, then you just need to install `grpcio` as Jupyter is already included in anaconda.
+
+Besides the basic function of python interpreter. You can use all the IPython advanced features as you use it in Jupyter Notebook.
```

Comment:
I think this should be one sentence
`Besides the basic function of python interpreter. You`
=>
`Besides the basic function of python interpreter, you`
or\
 =>
`In addition to all basic functions of the python interpreter, you`

Reply:
Fixed